code/AEPSDK.brs

```diff
@@ -297,8 +297,11 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
                 _adb_logError("createMediaSession() - Cannot create media session, invalid XDM data")
                 return
             end if
-
-            sessionId = m._private.mediaSession.startNewSession()
+            playhead = 0
```

Comment:
There is a position variable on L285, can we check if the playhead is valid there and combine this logic.

Reply:
The L285 code is used to handle the position/playhead of the previous session. It might be confusing to mix them together.

Reply:
We will get getting Playhead from customer right? So ideally we dont need any playhead logic. This is for fallback? What about sessionEnd?

Reply:
Discussed offline and decided to move this logic from the API layer to the SDK (task node) layer in another PR.